sections/featured-product.liquid

```diff
@@ -89,6 +89,16 @@
                   {%- endform -%}
                 </div>
               {%- endif -%}
+              {%- if shop.taxes_included or shop.shipping_policy.body != blank -%}
+                <div class="caption rte">
```

Comment:
When there are no product image, the text is left aligned. Can we align it center?
When there is a product image, it looks fine.

[![alt](https://screenshot.click/14-27-cy194-fwz3l.png)](https://screenshot.click/14-27-cy194-fwz3l.png)

Reply:
Good catch! Fixed https://github.com/Shopify/dawn/pull/629/commits/e5c9de9ce6e5626bddfa788e2112fcc937234fce